### ISSUE TITLE
Methods.java uses unchecked operation - says javac

### ISSUE THREAD

**Issue by LIttleAncientForestKami** [precol/MagicNumbers#2]
When compiling:

```
Note: Methods.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
```

FYI. Do you know why that happens?

**Comment by LIttleAncientForestKami** [comment]
> When it comes to 2nd mail, I think it was because I wrote ArrayList Instead of
> ArrayList<String[]>. 

Yes, AL without generifics gives that kind of warning. Do you know why?

**Comment by LIttleAncientForestKami** [comment]
Also, the warning still is there.

**Comment by precol** [comment]
> > When it comes to 2nd mail, I think it was because I wrote ArrayList Instead of
> > ArrayList<String[]>.
> 
> Yes, AL without generifics gives that kind of warning. Do you know why?

To my understanding it's because with no variable specified list can have added Strings, ints, Arrays and other types, which would be problematic if we would put different types in the same list. If we specify the type compiler knows which type it should accept so we eliminate that problem.

**Comment by precol** [comment]
I tried to compile with Xlint, but couldn't manage it both through terminal and Intellij to work. I think I got rid of the problem, but it might be the case that I didn't look enough. I'll try some more, but I didn't have any notes about using unchecked operations and didn't have a FNFE. If program still isn't working properly I can work on it more, however at the moment I can't say what exactly causes problems.
